src/test/java/com/ericsson/gerrit/plugins/eiffel/handlers/DataBaseHandlerTest.java

```diff
@@ -0,0 +1,212 @@
+
+        String eventId = dbHandler.getEventID(Table.SCS_TABLE, BRANCH);
+        assertFalse(firstEiffelEventId.equals(eventId));
+        assertEquals("Expect fetched event ID", secondEiffelEventId, eventId);
+    }
+
+    /**
+     * Insert should fail if value and key already exist
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testInsertIntoExistingFails() throws Exception {
+        String firstEiffelEventId = generateEiffelEventId();
+        String secondEiffelEventId = generateEiffelEventId();
+        dbHandler.insertInto(Table.SCS_TABLE, BRANCH, firstEiffelEventId);
+
+        try {
+            dbHandler.insertInto(Table.SCS_TABLE, BRANCH, secondEiffelEventId);
+            assertEquals("Expected the call to throw and Exception but none was thrown!", true, false);
+        } catch (SQLException e) {
+            // test passed!
+        }
+
+    }
+
+    /**
+     * Empty db should return empty string
+     *
+     * @throws Exception
+     */
+    @Test(expected = NoSuchElementException.class)
+    public void testGetNoneExistingEventIdReturnsNull() throws Exception {
+        dbHandler.getEventID(Table.SCS_TABLE, FAULTY_BRANCH);
+    }
+
+    /**
+     * Test Table enum
+     */
+    @Test
+    public void testTables() {
+        Table scsTable = Table.SCS_TABLE;
+        Table sccTable = Table.SCC_TABLE;
+
+        assertEquals("Table.SCS_TABLE tabler key should be", SCS_TABLE_KEY, scsTable.getKeyName());
+        assertEquals("Table.SCC_TABLE tabler key should be", SCC_TABLE_KEY, sccTable.getKeyName());
+
+    }
+
+    /**
+     * Test throwing several exceptions and ensure they cause the class to return
+     * correct values and or the exception is correctly caught. Exceptions caused by
+     * mocks should not leak into the test.
+     *
+     * @throws Exception
+     */
+    @Test(expected = NoSuchElementException.class)
+    public void testExceptionsIsThrown() throws Exception {
+        // Prepare mocks
+        Connection connection = mock(Connection.class);
+        PreparedStatement preparedStatement = mock(PreparedStatement.class);
+        Statement statement = mock(Statement.class);
+        ResultSet result = mock(ResultSet.class);
+
+        PowerMockito.mockStatic(DriverManager.class);
+        BDDMockito.given(DriverManager.getConnection(Mockito.any())).willReturn(connection);
+
+        Mockito.doReturn(preparedStatement).when(connection).prepareStatement(Mockito.any());
+        Mockito.doReturn(result).when(preparedStatement).executeQuery();
+
+        // When mocking an exception while executing getEventID the function should
+        // return an empty String,
+        // exception may be thrown when no values was found and should be empty.
+        Mockito.when(result.next()).thenThrow(new SQLException("Exception thrown by test"));
+
+        exception.expect(RuntimeException.class);
+        dbHandler.getEventID(Table.SCS_TABLE, BRANCH);
```

Comment:
Tetsed this.
The test concider it self completed at line 166 and nothing below is tested.